components/_patterns/00-utilities/_reusable-classes.scss

```diff
@@ -132,42 +132,34 @@ img.align-center {
 
 //Padding on all sides (equal)
```

Comment:
Adding padding changes the layout. Can we move these padding classes into the layout.scss file?